WeakAuras/WeakAuras.lua

```diff
@@ -3412,6 +3414,47 @@ function WeakAuras.Modernize(data)
     end
   end
 
+  if data.internalVersion < 22 then
+    if data.regionType == "aurabar" then
+      local border = {
+        ["type"] = "subborder",
+        border_visible = data.border,
+        border_color = data.borderColor,
+        border_edge = data.borderEdge,
+        border_offset = data.borderOffset,
+        border_size = data.borderSize,
+      }
+
+      data.border = nil
+      data.borderColor = nil
+      data.borderEdge = nil
+      data.borderOffset = nil
+      data.borderInset = nil
+      data.borderSize = nil
+      data.backdropColor = nil
```

Comment:
I wonder whether we should leave the backdrop related data in aurabar. If there are users that make use of backdrops, we can readd backdrops and have the right data